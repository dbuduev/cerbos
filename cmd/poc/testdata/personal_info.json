{
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "type": "object",
  "properties": {
    "first_name": { "type": "string" },
    "last_name": { "type": "string" },
    "phones": { "type": "array", "items": {"type":  "string"} },
    "address": {
      "type": "object",
      "properties": {
        "street_address": { "type": "string" },
        "city": { "type": "string" },
        "state": { "type": "string" }
      }
    }
  },
  "required": ["street_address", "city", "state"]
}
