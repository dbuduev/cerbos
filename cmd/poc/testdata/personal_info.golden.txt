class PersonalInfo {
  address: PersonalInfoAddress;
  first_name: string;
  last_name: string;
  phones: Array<string>;
}

class PersonalInfoAddress {
  city: string;
  state: string;
  street_address: string;
}
